pysteps/__init__.py

```diff
@@ -21,6 +21,10 @@
 from . import verification as vf
 from . import visualization as plt
 
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I'm following `pylint` suggestions but I'm probably wrong.
I'm changing to lowercase with the next commit.

Reply:
Pylint expects that global variables should be uppercase.  However, we try to follow [a different naming convention](https://pysteps.readthedocs.io/en/latest/developer_guide/contributors_guidelines.html#coding-style-summary).

Reply:
Maybe this can be another issue but: why not implement a `pylintrc` file in the root of the project, add to travis and enforce all code to use it?
You mention _Google’s python style guide_: it give you a good start point with [pylintrc](https://github.com/google/styleguide/blob/gh-pages/pylintrc).